synquacer.mk

```diff
@@ -0,0 +1,131 @@
+################################################################################
+# Following variables defines how the NS_USER (Non Secure User - Client
+# Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
+# S_USER (Secure User - TA) are compiled
+################################################################################
+COMPILE_NS_USER   ?= 64
+COMPILE_NS_KERNEL ?= 64
+COMPILE_S_USER    ?= 64
+COMPILE_S_KERNEL  ?= 64
+
+DEBUG=0
+ifeq ($(DEBUG),1)
+ARM_TF_BUILD ?= debug
+else
+ARM_TF_BUILD ?= release
+endif
+ifeq ($(DEBUG),1)
+EDK2_BUILD ?= DEBUG
+else
+EDK2_BUILD ?= RELEASE
+endif
+
+################################################################################
+# Includes
+################################################################################
+include common.mk
+include toolchain.mk
+
+################################################################################
+# Paths to git projects and various binaries
+################################################################################
+ARM_TF_PATH ?= $(ROOT)/arm-trusted-firmware
+ARM_TF_OUT ?= $(ARM_TF_PATH)/build/synquacer/$(ARM_TF_BUILD)
+ARM_TF_BL31 ?= $(ARM_TF_OUT)/bl31.bin
+ARM_TF_BL32 ?= $(ARM_TF_OUT)/bl32.bin
+ARM_TF_FIP ?= $(ARM_TF_OUT)/fip_all_arm_tf.bin
+FIPTOOL ?= $(ARM_TF_PATH)/tools/fiptool/fiptool
+EDK2_PATH ?= $(ROOT)/edk2
+EDK2_PLATFORMS_PATH ?= $(ROOT)/edk2-platforms
+EDK2_NON_OSI_PATH ?= $(ROOT)/edk2-non-osi
+EDK2_PKGS_PATH := "$(EDK2_PATH):$(EDK2_PLATFORMS_PATH):$(EDK2_NON_OSI_PATH)"
+EDK2_FIP ?= $(EDK2_NON_OSI_PATH)/Platform/Socionext/DeveloperBox/fip_all_arm_tf.bin
+
+################################################################################
+# Targets
+################################################################################
+.PHONY: all
+all: arm-tf edk2 optee-os
+
+.PHONY: clean
+clean: arm-tf-clean edk2-clean optee-os-clean
+
+################################################################################
+# ARM Trusted Firmware
```

Comment:
I know we still have "ARM Trusted Firmware" in already existing makefiles, but the since the project changed the name a while ago, the correct name is actually "Trusted Firmware A". Because of that, please also change all `ARM_TF_...` defines to `TFA_...` instead.

Reply:
got it. I will rename them.